File: RuntimeData/runtime/app/src/main/java/Params/PARAM_ZONE.java
/* Copyright (c) 1996-2013 Clickteam
 *
 * This source code is part of the Android exporter for Clickteam Multimedia Fusion 2.
 * 
 * Permission is hereby granted to any person obtaining a legal copy 
 * of Clickteam Multimedia Fusion 2 to use or modify this source code for 
 * debugging, optimizing, or customizing applications created with 
 * Clickteam Multimedia Fusion 2.  Any other use of this source code is prohibited.
 *
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 * IN THE SOFTWARE.
 */
//----------------------------------------------------------------------------------
//
// PARAM_ZONE: zone a l'ecran
//
//----------------------------------------------------------------------------------
package Params;

import Application.CRunApp;

public class PARAM_ZONE extends CParam
{
    public short x1;
    public short y1;
    public short x2;
    public short y2;
    
    @Override
	public void load(CRunApp app) 
    {
        x1=app.file.readAShort();
        y1=app.file.readAShort();
        x2=app.file.readAShort();
        y2=app.file.readAShort();
    }
    
}

File: RuntimeData/xnaruntime/RuntimeXNA/RuntimeXNA/Params/PARAM_ZONE.cs
//----------------------------------------------------------------------------------
//
// PARAM_ZONE: zone a l'ecran
//
//----------------------------------------------------------------------------------
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using RuntimeXNA.Application;
using RuntimeXNA.Services;

namespace RuntimeXNA.Params
{	
	public class PARAM_ZONE:CParam
	{
		public short x1;
		public short y1;
		public short x2;
		public short y2;
		
		public override void  load(CRunApp app)
		{
			x1 = app.file.readAShort();
			y1 = app.file.readAShort();
			x2 = app.file.readAShort();
			y2 = app.file.readAShort();
		}
	}
}